easybuild/framework/easyblock.py

```diff
@@ -455,12 +455,16 @@ def fetch_patches(self, patch_specs=None, extension=False, checksums=None):
         All patches will be checked if a file exists (or can be located)
         """
         if patch_specs is None:
-            patch_specs = self.cfg['patches']
+            post_install_patches = self.cfg['postinstallpatches']
+            patch_specs = self.cfg['patches'] + post_install_patches
+        else:
+            post_install_patches = []
 
         patches = []
         for index, patch_spec in enumerate(patch_specs):
 
             patch_info = create_patch_info(patch_spec)
+            patch_info['postinstall'] = patch_spec in post_install_patches
```

Comment:
`patch_info['postinstall']` will only be true for post-install patches whenever `fetch_patches()` is called with `patch_specs=None`, but not when it will be called with some specific `patch_spec`. Is that ok?

Reply:
`fetch_patches` is only passed fed `patch_specs` when fetching patch files for extensions, so it's fine if `postinstall` is always `False` then.

That's not really a limitation, since you can always use ` postinstallpatches` to patch any installed extensions.

Reply:
thanks for the clarification :+1: